src/main/java/seedu/address/ui/TuitionCard.java

```diff
@@ -1,62 +1,55 @@
 package seedu.address.ui;
 
-import java.time.format.DateTimeFormatter;
-
+import javafx.collections.FXCollections;
+import javafx.collections.ObservableList;
 import javafx.fxml.FXML;
 import javafx.scene.control.Label;
-import javafx.scene.layout.HBox;
+import javafx.scene.control.ListCell;
+import javafx.scene.control.ListView;
 import javafx.scene.layout.Region;
+import javafx.scene.layout.VBox;
 import seedu.address.model.session.Session;
 import seedu.address.model.student.Student;
-import seedu.address.model.tuition.Tuition;
 
 /**
- * An UI component that displays information of a {@code Tuition}.
+ * An UI component that displays information of a {@Student} and a list the {@code Session} of that {@code Student}.
  */
 public class TuitionCard extends UiPart<Region> {
 
     private static final String FXML = "TuitionListCard.fxml";
```

Comment:
Is this perhaps `TuitionCard.fxml` for consistency?

Reply:
Kinda followed the original StudentCard from AB3.
![image](https://user-images.githubusercontent.com/39006568/112808710-dd355780-90ab-11eb-9d2a-2ce30450d79d.png)
I think we just keep it? 😅